librz/util/list.c

```diff
@@ -13,16 +13,41 @@ RZ_API void rz_list_iter_free(RzListIter *list) {
 	/* do nothing? */
 }
 
+/**
+ * \brief returns the next RzList iterator in the list
+ *
+ * */
 RZ_API RzListIter *rz_list_iter_get_next(RzListIter *list) {
 	rz_return_val_if_fail(list, NULL);
 	return list->n;
 }
 
+/**
+ * \brief returns the value stored in the next RzList iterator
+ *
+ * */
```

Comment:
```suggestion
 **/
```

Reply:
i think that is added by clang-format.

Reply:
weird, unless they have changed something very recently. I think it should be `/** **/`